### PR TITLE
Prine-134 Add pi status to admin tool

### PR DESCRIPTION
**Pull request by davidheimgartner**
Run "yarn install" to install luxon

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,6 +35,7 @@
     "core-js": "3.6.5",
     "express": "4.17.1",
     "express-force-https": "1.0.0",
+    "moment": "^2.27.0",
```
Comment (hirsch88) [inline package.json:38]: moment is a huge lib and most of the time we only need like 1 - 5 % of it. there is smaller and faster brother of moment called luxon
https://moment.github.io/luxon/

File: src/app/pages/admin/AdminDashboard.vue [hunk 2]
```diff
@@ -82,3 +90,20 @@ export default defineComponent({
   },
 })
 </script>
+
+<style lang="scss" scoped>
+
+.gateControlDeviceContainer {
```
Comment (hirsch88) [inline src/app/pages/admin/AdminDashboard.vue:96]: Looks like duplicated code :-)
try to create a component for that

**Review by hirsch88 — CHANGES_REQUESTED** [review]

File: src/app/pages/admin/AdminDashboard.vue (hunk 2 above)
Comment (prine) [reply src/app/pages/admin/AdminDashboard.vue]: Oops, I just forgot to delete the css. It is already an external component-

File: package.json (hunk 1 above)
Comment (prine) [reply package.json]: We have exchanged moment with luxon

**Pull request merged**